src/pages/MainPage.vue

```diff
@@ -320,17 +320,14 @@ export default {
       searchInput: '',
       showSearch: false,
       classInfoStack: [],
-      currentSemester: 0,
+      currentSemester: 1,
```

Comment:
This is definitely good for non-logged in users, but FireRoad still has default of 0, so unless we change incoming 0's into 1's, users who have never changed their year will still have Fall 2019 show up as the first year.  If we want to implement this it would go in the verify function after you receive the current semester on the FireRoad server.

Reply:
Good call. I put it inside `setSemester` to be safe